api/controllers/users.controller.js

```diff
@@ -0,0 +1,72 @@
+const db = require("../models");
+const { hashHelpers, jwtHelpers } = require('../helpers');
+const errors = require('../services/errorHandlers/index');
+
+const signup = async (req, res) => {
+
+    const emailExists = await db.users.findOne({ where: { email: req.body.email } });
+    const usernameExists = await db.users.findOne({ where: { username: req.body.username } });
+
+    if(emailExists || usernameExists) {
+        throw new errors.userAlreadyExistsError();
+    }
+
+    const hashedPassword = await hashHelpers.createHash(req.body.password);
+    const transaction = await db.sequelize.transaction();
+
+    const user = {
+        id: req.body.id,
+        username: req.body.username,
+        email: req.body.email,
+        password: hashedPassword,
+    }
+
+    db.users.create(user, { transaction })
+        .then(data => {
+            res.send({ message: "You've been successfully signed up!" });
+            transaction.commit();
+        })
+        .catch(err => {
+            transaction.rollback();
+            throw new errors.signupFailedError();
+        })
+};
```

Comment:
Change the name to camelCase (signIn). Take out business logic from controllers to services. Also you can return a token from signIn so the user will be successfully authenticated just after it.

In controllers you should get the request data, validate it, call the services and return the result.

Also try to use same indents at project (I recommend 2 spaces)